src/pages/Dashboard/Settings.tsx

```
@@ -5,10 +5,95 @@ import {
+import checkIcon from "../../assets/images/checkIcon.png"
+import { useNavigate } from "react-router-dom";
 const Settings = () => {
   const [activeTab, setActiveTab] = useState("details");
-  const [withdrawalMethod, setWithdrawalMethod] = useState("");
+  const [oldPassword,setOldPassword] = useState("")
+  const [newPassword, setNewPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+  const [feedbackType, setFeedbackType] = useState("")
+  const [message,setMessage] = useState("")
+  const [showModal,setShowModal] = useState(false);
+  const [changepassword,{isLoading}] = useChangePasswordMutation() as any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)